tests/rustdoc-ui/track-diagnostics.rs

```diff
@@ -7,4 +7,5 @@
 
 struct A;
 struct B;
-const S: A = B;
+
+pub const S: A = B;
```

Comment:
A side-effect of this is that consts are only required to typeck if they're being documented.

Reply:
*in rustdoc*?

Reply:
oh 💀 yes in roblox